src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -112,6 +123,12 @@ export class CalciteAlert {
     this.alertLinkEl = this.el.querySelectorAll("calcite-link")[0] as HTMLCalciteLinkElement;
   }
 
+  disconnectedCallback(): void {
+    if (this.autoDismissTimeout) {
+      window.clearTimeout(this.autoDismissTimeout);
```

Comment:
I missed one more, you can clear the timeout without checking if it exists. This will end in a noop if there is no associated timeout ID.

Reply:
nice catch 🏈